components/http_foundation.rst

```diff
@@ -420,12 +420,12 @@ conditional value specified in the client Request, use the
 :method:`Symfony\\Component\\HttpFoundation\\Response::isNotModified`
 method::
 
-    if ($response->isNotModified($request)) {
+    if (!$response->isNotModified($request)) {
```

Comment:
> hm, ok. The documentation is still pretty confusing as written, though, because it presents one case (request not modified), but doesn't actually suggest what to do if the response is modified, because the response still needs to be sent.

That part is omitted because you will then have to populate the response body as usual. But if you think that this could be clarified, we could expand the example and add comments explaining this.